EIPS/eip-4444.md

```diff
@@ -0,0 +1,101 @@
+---
+eip: 4444
+title: Bound Historical Data in Execution Clients
+description: Prune historical data in clients older than one year
+author: George Kadianakis (@asn_d6), lightclient (@lightclient), Alex Stokes (@ralexstokes)
+discussions-to: TBD
+status: Draft
+type: Standards Track
+category: Core
```

Comment:
I think this should go in `Networking` rather than `Core`.  Also, it should include details on what the expected response should be to a request for blocks older than the save duration.

Reply:
Given the definition of a `Core` EIP from EIP-1, I feel like this fits perfectly in the "non-consensus critical change, but relevant to core dev discussions". This EIP affects networking, RPC interface, and the possibility of full validating the chain.

> Core: improvements requiring a consensus fork (e.g. EIP-5, EIP-101), as well as changes that are not necessarily consensus critical but may be relevant to “core dev” discussions (for example, [EIP-90], and the miner/node strategy changes 2, 3, and 4 of EIP-86)

Reply:
EIP-90 isn't an EIP... we should fix that example... I think functionally, all Core EIPs are consensus changes.  I would rather fix EIP-1 to reflect that then use what it says to allow a single EIP that doesn't follow the pattern that every other EIP (I think, at least everything in the last couple years) currently follows.

Reply:
Meh fair. It still *feels* like a core change because it affecting many things like networking, RPC, and syncing trust assumptions. But I've updated it to `Networking` for now.